### PR TITLE
[RFC] add a default crud controller

### PR DESCRIPTION
**Pull request by docteurklein**

### CODE REVIEW

File: features/crud-controller.feature [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Feature: Default CRUD controller
+    In order to provide default CRUD operations
+    As a developer
+    I need rad bundle to provide a default CRUD controller
+
+    Scenario: use provided controller
+        Given I write in "App/Resources/config/routing.yml":
+        """
+        App:Foo:
+            controller: app.rad_bundle.controller.crud_controller
```
Comment (docteurklein) [inline features/crud-controller.feature:10]: service controller friendly

File: features/crud-controller.feature [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            defaults:
+                _view: App:Foo:show
```
Comment (docteurklein) [inline features/crud-controller.feature:12]: you must pass the view since it can't guess it from the controller name

**Comment by ghost** [comment]
is there any way to share work here with sylius resourcebundle?

**Comment by docteurklein** [comment]
Sylius resource bundle is a slightly different approach, but it could be possible to use some parts of it yes.

**Comment by einenlum** [comment]
This bundle is not maintained anymore.

**Pull request closed without merging**